.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
```suggestion
<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.-->
```
Sorry, I forgot to say this before in the chat!! Could we make this into a comment instead, in case someone forgets to delete this bit, it wont show up in the PR!